packages/vue-component-library/src/lib-components/FlexibleBlocks.vue

```diff
@@ -221,9 +222,28 @@ function getComponent(name) {
   lang="scss"
   scoped
 >
+// default theme
 .flexible-blocks {
   .more-information {
     @include visually-hidden;
   }
 }
+
+// ftva theme
+.ftva.flexible-blocks {
+  .flexible-block-section-wrapper {
+    // sections within flexible blocks have bold titles and medium grey summaries
+    :deep(.section-header) {
+      margin-bottom: 12px;
+      .section-title {
+          @include ftva-h5;
+          color: $accent-blue;
+        }
+      .section-summary {
+        @include ftva-body;
```

Comment:
@farosFreed A little detail: In figma, the section-summary's token is `ftva-body-1`. Unless `ftva-body` and `ftva-body-1` are the same things.